notebooks/1_compute_embeddings_weaviate_nl.ipynb

```diff
@@ -311,7 +311,7 @@
     "    \"duurzaam\",\n",
     "    \"milieu\",\n",
     "]  # Search term(s) for filtering the corpus. If empty, it will only create Passages for the pieces of text that match the FILTER_TERMS\n",
-    "FILTER_TERMS = open(\"sustainability-filter-words.txt\").read().split(\"\\n\")\n",
+    "FILTER_TERMS = open(\"seed_terms/sustainability-filter-words.txt\").read().split(\"\\n\")\n",
```

Comment:
Consider moving the `seed_terms` to the `tempo_embeddings/data` directory. You can then import the `DATA_DIR` variable from `settings`.